.gitpod.yml

```diff
@@ -0,0 +1,29 @@
+tasks:
+  - init: |
+      # remove once latest maven works again or we switch to maven wrapper
+      sed -i 's/sdkman_auto_answer=false/sdkman_auto_answer=true/' $HOME/.sdkman/etc/config
```

Comment:
You may consider to introduce a custom Dockerfile instead [[1](https://www.gitpod.io/docs/config-docker#configure-a-custom-dockerfile)]

Reply:
Does it make much of a difference? It will still have to build the docker image anyway.

I tried this last week sometime with the dockerfile and it didn't work right for me, but I probably missed something.

https://github.com/timja/jenkins/blob/b277277379722a1c8bdbdc61f675817c2bb89f27/.gitpod.yml#L17-L18

Reply:
It is a bit different from point of caching. Gitpod has 2 builds one for changes in Dockerfile and other in prebuild for changes in repo. By moving tools setup which is not dependent on code in repo to Dockerfile you decrease time of prebuilds for all branches.

Reply:
It should be like:
```yaml
image:
  file: .gitpod.Dockerfile
```

Reply:
that worked, I think I was missing setting maven to be the default version before